### ISSUE TITLE
Using H5Spark in Jupyter

### ISSUE THREAD

**Issue by valiantljk** [valiantljk/h5spark#6]
A user requests to use H5Spark in Jupyter, which is a popular notebook used anywhere. 
The current issue is that the class path is not recognized in the notebook, i.e., the h5spark library can not be exported and used.

**Comment by eracah** [comment]
Yes. This is a strange issue that @valiantljk  and I were dealing with. I suspect it's some issue with PYTHONPATH and **init**.py Let me look into it

**Comment by jakehockey10** [comment]
May I ask what the current status of this issue is?

**Comment by valiantljk** [comment]
Hi Evan, 
Is h5spark usable in Jupyter?

Best,
Jialin

**Comment by eracah** [comment]
I believe so. For python at least. If you can get a pyspark notebook set up then you can do it. All you need is the h5spark python module to be on your PYTHONPATH or your sys.path

I will note that I no longer contribute to h5spark, so I cannot guarantee bug-free python code for h5spark

Thanks,
Evan

**Comment by valiantljk** [comment]
Sounds good. Thanks and good luck.

-Jialin
